Flutter/calendar/appointments.md

```diff
@@ -408,6 +408,47 @@ class _AppointmentDataSource extends CalendarDataSource {
 
 ![Adding recurrence appointment](images/appointments/recurrence-appointment.png)
 
+### Recurrence appointments on last day of month
```

Comment:
Add this topic under **Flexible working days and working hours**

Reply:
I have checked and I have updated the topic (number of days in view) is under flexible working days and working hours. I have updated my changes in latest commit.